qa-tests-backend/src/test/groovy/ImageManagementTest.groovy

```diff
@@ -279,7 +296,11 @@ class ImageManagementTest extends BaseSpecification {
 
         cleanup:
         // Should be able to call this multiple times safely in case of any failures previously
-        CVEService.unsuppressCVE("CVE-2010-0928")
+        if (Env.CI_JOBNAME.contains("postgres")) {
+            CVEService.unsuppressImageCVE(cve)
+        } else {
+            CVEService.unsuppressCVE(cve)
+        }
```

Comment:
Can we just push this `Env.CI_JOBNAME` check into the `unsuppressImageCVE` function, rather than repeating it everywhere?